test/tests/functions/baselinecorrection/half_sine_scaled.i

```diff
@@ -0,0 +1,41 @@
+# A test of the BaselineCorrection function output scaled by an input factor.
```

Comment:
Or maybe just include half sine scaled and consider removing half sine and full sine.

Reply:
@somu15 what if I put all of these tests into a single input file? They could all be operated on as separate function objects, and be written to separate post-processors.